frontend/src/lib/forms/UserTypeSelect.svelte

```diff
@@ -18,7 +18,8 @@
 </script>
 
 <div class="relative">
-  <Select id="type" label={$t('admin_dashboard.user_filter.user_type.label')} bind:value {...rest}>
+  <!-- Note: important that ...rest be at the start of the attributes list so that `label={$t(...)}` will not be overridden by the `label` prop in ...rest -->
```

Comment:
For next time, you could also solve this with: `interface Props extends Omit<SelectProps, 'label'>`.

EDIT: actually, that's probably what you want to do to fix all the remaining eslint errors

Reply:
Weird that I'm not getting any lint errors when I run `pnpm lint`. I'll go ahead and do `Omit<..., 'label'>` everywhere anyway, but I might not be able to see the lint errors get fixed until after pushing. So it's possible I'll be pushing a few commits here...

Reply:
Done in commit ed306c4f7.

Reply:
Oh, they were svelte check errors, not lint errors.